src/Components/Search/SearchBar.tsx

```diff
@@ -175,8 +178,33 @@ export class SearchBar extends Component<Props, State> {
     )
   }
 
+  reportPerformanceMeasurement = () => {
+    const duration = performance.now() - this.performanceStart
+    const deviceType = sd.IS_MOBILE ? "mobile" : "desktop"
+
+    const payload = {
+      type: "timing",
+      name: "autocomplete-search-response",
+      timing: duration,
+      tags: [`device-type:${deviceType}`, "design:rich"],
+    }
+
+    request.post(sd.VOLLEY_ENDPOINT).send({
+      serviceName: "force",
```

Comment:
#minor this is sort of hardcoded (which I think is fine!), but possibly `VOLLEY_SERVICE_NAME` should be an env var.

Reply:
Good point. Is that something that needs to be added to Force first? Maybe a reasonable follow-up PR?